command_line/stills_process.py

```diff
@@ -193,11 +193,10 @@ def do_import(filename):
     raise Abort("Got multiple datablocks from file %s"%filename)
 
   # Ensure the indexer and downstream applications treat this as set of stills
-  from dxtbx.imageset import ImageSet
   reset_sets = []
 
   for imageset in datablocks[0].extract_imagesets():
-    imageset = ImageSet(imageset.data(), imageset.indices())
+    imageset = imageset.__class__(imageset.data(), imageset.indices())
```

Comment:
I'm not sure, I like calling imageset.__class__  like this, perhaps we could add a classmethod factory function to imageset that instantiates a new imageset of that class. Or something like that.